api/server/router/grpc/grpc.go

```diff
@@ -21,7 +21,7 @@ type grpcRouter struct {
 // NewRouter initializes a new grpc http router
 func NewRouter(backends ...Backend) router.Router {
 	unary := grpc.UnaryInterceptor(grpc_middleware.ChainUnaryServer(unaryInterceptor(), grpcerrors.UnaryServerInterceptor))
-	stream := grpc.StreamInterceptor(grpc_middleware.ChainStreamServer(otelgrpc.StreamServerInterceptor(), grpcerrors.StreamServerInterceptor))
+	stream := grpc.StreamInterceptor(grpc_middleware.ChainStreamServer(otelgrpc.StreamServerInterceptor(), grpcerrors.StreamServerInterceptor)) //nolint:staticcheck // TODO(thaJeztah): ignore SA1019 for deprecated options: see https://github.com/moby/moby/issues/47437
```

Comment:
FWIW; PR that removes the deprecated calls is https://github.com/moby/moby/pull/47919

Reply:
Oh nice, although that might require a buildkit upgrade to v0.14.0 for some of the utility package updates for tracing.

Reply:
Oh! Good one; didn't remember that those may be needed (or at least, I didn't check if those were _required_ for that patch, or if there's alternatives).

In either case; the deprecated ones still work (at least for now) so not critical for sure!